internal/cli/atlas/kubernetes/config/generate.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"context"
+	"strings"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/kubernetes/operator"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type GenerateOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	clusterName     string
+	orgID           string
+	includeSecrets  bool
+	targetNamespace string
+	store           store.AtlasOperatorGenericStore
+	credsStore      store.CredentialsGetter
+}
+
+func (opts *GenerateOpts) initStores(ctx context.Context) func() error {
+	return func() error {
+		var err error
+
+		profile := config.Default()
+		opts.store, err = store.New(store.AuthenticatedPreset(profile), store.WithContext(ctx))
+		if err != nil {
+			return err
+		}
+		opts.orgID = profile.OrgID()
+		opts.credsStore = profile
+
+		return nil
+	}
+}
+
+func normalizeClustersNames(input string) []string {
+	if input == "" {
+		return []string{}
+	}
+	clusters := strings.Split(input, ",")
+	if len(clusters) == 0 {
+		return []string{input}
+	}
+	return clusters
+}
+
+func (opts *GenerateOpts) Run() error {
+	result, err := operator.NewConfigExporter(opts.store, opts.credsStore, opts.ProjectID, opts.orgID).
+		WithClustersNames(normalizeClustersNames(opts.clusterName)).
+		WithTargetNamespace(opts.targetNamespace).
+		WithSecretsData(opts.includeSecrets).
+		Run()
+	if err != nil {
+		return err
+	}
+	return opts.Print(result)
+}
+
```

Comment:
[nit] we generally have a comments before the builder that shows the format of the command

Reply:
Thanks! I added one